apps/backoffice-v2/src/lib/blocks/components/KycBlock/hooks/useKycBlock/useKycBlock.tsx

```diff
@@ -47,11 +47,15 @@ export const useKycBlock = ({
   const { noAction } = useCaseDecision();
   const kycSessionKeys = Object.keys(childWorkflow?.context?.pluginsOutput?.kyc_session ?? {});
 
-  const { documents, isLoading: isLoadingDocuments } = useDocumentsAdapter({
+  const { documents: allDocuments, isLoading: isLoadingDocuments } = useDocumentsAdapter({
     entityIds: [childWorkflow?.context?.entity?.ballerineEntityId ?? ''],
     documents: childWorkflow?.context?.documents ?? [],
   });
 
+  const documents = useMemo(() => {
+    return allDocuments?.filter(document => document.type === 'identification_document') ?? [];
```

Comment:
documents already somewhat implies it's all documents. It's still just the documents for the specific entity id as well.
```suggestion
  const nonIdentificationDocuments = useMemo(() => {
    return allDocuments?.filter(document => document.type === 'identification_document') ?? [];
```

Reply:
KYC now renders only identification documents.